src/runtimes/node/bundler.js

```diff
@@ -106,7 +106,10 @@ const bundleJsFile = async function ({
       warnings,
     }
   } catch (error) {
-    error.customErrorInfo = { type: 'functionsBundling', location: { functionName: name, runtime: RUNTIME_JS } }
+    error.customErrorInfo = {
+      type: 'functionsBundling',
+      location: { bundler: JS_BUNDLER_ESBUILD, functionName: name, runtime: RUNTIME_JS },
```

Comment:
Do we want to show the bundler and/or the runtime in the error message title?

https://github.com/netlify/build/blob/a6ab7b9dbfebd21bcedcb5dbada4d1620cd6192a/packages/build/src/error/type.js#L86

Please note we do show the `location` fields in the error details, which is printed in the build logs. Also, the `location` fields are always sent to Bugsnag. Just wondering what you think about the error message title itself (which is shown in the error header and in the deploy summary).

Reply:
I personally don't think we need to show the runtime in the message because it probably offers less clarity than the noise it introduces. I think that field is useful mostly for Bugsnag. 

However, on that link I do see that we show `functionName`, which means that the field is mandatory. We don't have the function name when using esbuild from the ZISI flow. Adding it will mean passing that parameter around in a lot of functions, which I'm not super excited about, but it'll have to be done. 

I'll add to this PR.

Reply:
Makes sense! :+1:

Reply:
Done in https://github.com/netlify/zip-it-and-ship-it/pull/700/commits/94c32481a17aaa73083b65a8691be483d0cef689. Some of these functions are called recursively, and without types it's a bit difficult to ensure all cases are covered, but I think I've done it!

Reply:
Looks good!

Reply:
Thanks for your help! 🙌🏻